features/FEATURE_BLE/targets/TARGET_CORDIO/doc/PortingGuide.md

```diff
@@ -97,40 +84,37 @@ private:
 } // namespace ble
 ```
 
-It shall inherits publicly from the base class `CordioHCITransportDriver`. 
+It inherits publicly from the base class `CordioHCITransportDriver`.
+
+##### Functions
+
+* **Initialization/termination**: The functions `initialize` and `terminate` are responsible for initializing and terminating the transport driver. It is not necessary to initialize the transport in the constructor.
 
-* **Initialization/termination:** The functions `initialize` and `terminate` are 
-responsible for initializing and terminating the transport driver. It is not 
-necessary to initialize the transport in the constructor.
+* **Sending data**: The function `write` sends data as input to the Bluetooth controller and returns the number of bytes in the `data` buffer sent. Depending on the type of transport you implement, you may need to send the packet `type` to the controller before the packet data.
 
-* **Sending data** The function `write` shall sends data in input to the 
-Bluetooth controller and return the number of bytes in the `data` buffer sent. 
-Depending on the type of transport being implemented, the packet `type` might 
-have to be sent to the controller before the packet data.
+* **Receiving data**: Inject HCI data from the Bluetooth controller to the system by calling `on_data_received`. This is a static function provided by the base class.
 
-* **Receiving data**: HCI data from the Bluetooth controller shall be injected 
-in the system by invoking the function `on_data_received`. This function is 
-a static one and is provided by the base class. Its prototype is: 
+**Example:**
 
-```c++
+```
 void on_data_received(uint8_t* data_received, uint16_t length_of_data_received);
 ```
 
-#### HCIDriver 
+#### HCIDriver
 
-The skeleton of driver is: 
+The HCIDriver template is:
 
-```c++
+```
```

Comment:
The C++ tag is useful here.

Reply:
OK, only problem is it makes stuff render weirdly while editing.